.pre-commit-config.yaml

```diff
@@ -82,10 +82,10 @@ repos:
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.7.9
     hooks:
-      - id: flake8
-        name: flake8 except __init__.py
-        exclude: /__init__\.py$
-        additional_dependencies: ["flake8-bugbear==19.8.0"]
+      # - id: flake8
```

Comment:
Please, never edit pre-commit files. They ensure that the format is followed properly according OCA rules